src/main/scala/se/lu/nateko/cp/data/formats/TimeSeriesParsing.scala

```diff
@@ -1,5 +1,16 @@
 package se.lu.nateko.cp.data.formats
 
+trait TextFormatParser[A <: StandardParsingAcculumator] {
+	def parseLine(acc: A, line: String): A
+}
+
+trait StandardParsingAcculumator extends ParsingAccumulator {
+	def header: Header
+	def cells: Array[String]
+}
+
+class Header(val colNames: Array[String], val formats: Array[Option[ValueFormat]], val error: Option[Throwable])
```

Comment:
Consider renaming this to `StandardHeader`, as `Header` class directly in `formats` package is a bit ambiguous.